implementation/core/src/main/java/io/smallrye/faulttolerance/core/bulkhead/BulkheadBase.java

```diff
@@ -4,55 +4,16 @@
 
 import io.smallrye.faulttolerance.core.FaultToleranceStrategy;
 
-/**
- * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
```

Comment:
:(

Reply:
I tend to do this when I do larger modifications to the classes, such that I'd have to add another author tag for myself. I find them an unnecessary clutter that convey exactly zero information. Your authorship is carefully preserved in Git history :-)